### ISSUE TITLE
Feature Request scale for steps in range slider

### ISSUE THREAD

**Issue by SumNeuron** [plotly/dash-core-components#578]
It would be nice if the steps in range slide could be scaled (e.g. log)

**Comment by alexcjohnson** [comment]
From @MM-Lehmann in https://github.com/plotly/dash/issues/1250:

> It would be nice to have the dcc.Slider (and similar controls?) accept a `log=int()` attribute which would then convert all numbers/marks/tooltips of the control by calculating `x**log`. Of course, the default value would be `log=1`

`x**log` would be power scaling, not logarithmic - anyway there are various scaling functions which may be valuable.

**Comment by MM-Lehmann** [comment]
> `x**log` would be power scaling, not logarithmic - anyway there are various scaling functions which may be valuable.
Yeah, I've been a bit confusing in my wording. That's what i meant, anyway.
In my mind, it would be easy to just attach a scaler function, similar to a serialiser/deserialiser which converts set values in both directions. This way, you wouldn't have to change much in the backend. That said, the js slider supports a logarithmic scaling option, maybe that can be leveraged?
